### DISCUSSION TITLE
Falcon model support

### DISCUSSION THREAD

**Discussion by janjanusek** [microsoft/onnxruntime-genai#1164] [Model support]
Hello cozld you please add Falcon3 support to the up coming release which is supposed to beat qwen2.5?

here's the model link https://huggingface.co/tiiuae/Falcon3-3B-Instruct

**Comment by kunal-vaishnavi** [comment]
Falcon-3 is already supported as it uses the [LLaMA architecture](https://huggingface.co/tiiuae/Falcon3-3B-Instruct/blob/eef6d2441fe6baf58512eab03c77e020fbdab3ea/config.json#L3).
